vendor/k8s.io/kubernetes/cmd/kube-apiserver/app/apiextensions.go

```diff
@@ -30,6 +30,10 @@ import (
 	"k8s.io/kubernetes/cmd/kube-apiserver/app/options"
 )
 
+func createAPIExtensionsServer(apiextensionsConfig *apiextensionsapiserver.Config, delegateAPIServer genericapiserver.DelegationTarget) (*apiextensionsapiserver.CustomResourceDefinitions, error) {
+	return apiextensionsConfig.Complete().New(delegateAPIServer)
```

Comment:
If I trust my eyes, this is only an (unnecessary) move?

Reply:
> If I trust my eyes, this is only an (unnecessary) move?

fixed